### PR TITLE
feat: begin supporting the new rfc editor model

### PR DESCRIPTION
**Pull request by rjsparks**
This has one thing that I don't like, but changing it will be a bigger lift. It exposes what had to date been an internal artifact: the group type name slug of `rfcedtyp` which now shows in the urls when following some of the menus.

### CODE REVIEW

File: ietf/doc/templatetags/active_groups_menu.py [hunk 1]
```diff
@@ -27,4 +27,4 @@ def active_groups_menu(flavor):
     return render_to_string(
         "base/menu_active_groups.html",
         {"parents": parents, "others": others, "flavor": flavor},
-    )
\ No newline at end of file
+    )
```
Comment (larseggert) [inline ietf/doc/templatetags/active_groups_menu.py:30]: This seems like it could be removed from the PR.

**Review by larseggert — APPROVED** [review]
Noe an expert here, but LGTM.

**Review by jennifer-richards — APPROVED** [review]
Looks good to me. Only nit is some copyright notices that haven't been updated.

File: ietf/templates/group/active_rfced.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
+{% extends "base.html" %}
+{# Copyright The IETF Trust 2015, All Rights Reserved #}
```
[the remark this answers is not in the document]
Comment (rjsparks) [reply ietf/templates/group/active_rfced.html:2]: Yes, there was an svn commit trigger that we have not yet replaced in github.

**Pull request merged**